app/topics/show-topic/show-topic.component.ts

```diff
@@ -17,6 +17,8 @@ export class ShowTopicComponent implements OnInit, OnDestroy {
   title = '';
   userCanDelete: boolean = false;
   userCanEditDetails: boolean = false;
+  statusChange: boolean = false;
+  showStatustoStudents: boolean = true;
```

Comment:
should be `showStatusToStudents`

But the status should be shown to the student anyway.
Maybe better: `displayStatusOptions`